### PR TITLE
Add GitKraken Client

### PR DESCRIPTION
**Pull request by IDisposable**
Adds development utility client for git repository management for GitHub/GitLab/BitBucket/etc.

### CODE REVIEW

File: config/applications.json [hunk 1]
```diff
@@ -727,6 +727,14 @@
 		"link": "https://desktop.github.com/",
 		"winget": "Git.Git;GitHub.GitHubDesktop"
 	},
+	"WPFInstallgitkrakenclient": {
+		"category": "Development",
+		"choco": "git;gitkraken",
```
Comment (Doozy134) [inline config/applications.json:732]: Is this supposed to just be "gitkraken"? i.e. no git;

Comment (Doozy134) [reply config/applications.json:732]: p.s. make sure you change the PR to merge into the test branch not main :)

Comment (IDisposable) [reply config/applications.json:732]: I was wondering about that... copied from other GIT clients... I can easily remove

Comment (IDisposable) [reply config/applications.json]: Removed `git;`

Comment (IDisposable) [reply config/applications.json]: Changed to merge to test branch

**Comment by IDisposable** [comment]
Updated to remove `git;` and point merge toward the *test* branch

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**